ProjectLighthouse.Servers.Website/Controllers/Moderator/ModerationRemovalController.cs

```diff
@@ -50,7 +56,10 @@ public async Task<IActionResult> DeleteComment(int commentId, [FromQuery] string
         UserEntity? user = this.database.UserFromWebRequest(this.Request);
         if (user == null) return this.Redirect("~/login");
 
-        CommentEntity? comment = await this.database.Comments.FirstOrDefaultAsync(c => c.CommentId == commentId);
+        CommentEntity? comment = await this.database.Comments
+            .Include(c => c.TargetUser)
+            .Include(c => c.TargetSlot)
```

Comment:
Is this confirmed to be working without issue, I don't know how EF handles including a nullable foreign key

Reply:
To my knowledge including a nullable foreign key works fine? I'll double check though

Reply:
I don't see why this wouldn't work, see below

https://github.com/LBPUnion/ProjectLighthouse/blob/571986c384d04903b89ee240c67e1be05b7e8a31/ProjectLighthouse.Servers.Website/Pages/NotificationsPage.cshtml.cs#L24-L25

https://github.com/LBPUnion/ProjectLighthouse/blob/571986c384d04903b89ee240c67e1be05b7e8a31/ProjectLighthouse/Types/Entities/Website/WebsiteAnnouncementEntity.cs#L20-L21